packages/next-auth/src/providers/mercado-libre.js

```diff
@@ -0,0 +1,48 @@
+/** @type {import(".").OAuthProvider} */
+export default function MercadoLibreProvider(options) {
+  const { clientId, redirectUri, clientSecret } = options;
+
+  return {
+    id: "mercado-libre",
+    name: "Mercado Libre",
+    type: "oauth",
+    version: "2.0",
+    responseType: "code",
+    authorization: {
+      url: "https://auth.mercadolibre.cl/authorization?response_type=code",
+      params: { grant_type: "authorization_code" },
+    },
```

Comment:
```suggestion
    authorization: "https://auth.mercadolibre.cl/authorization",
```

These are the defaults already.